Tests/SettingsPageTest.cs

```diff
@@ -0,0 +1,183 @@
+using System.Linq;
+using AngleSharp.Dom;
+using Blazored.Modal;
+using Bunit;
+using HotelPriceScout.Data.Interface;
+using HotelPriceScout.Pages;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Tests
+{
+    public class SettingsPageTest : TestContext
+    {
+        public SettingsPageTest()
+        {
+            // Inject test services
+            Services.AddBlazoredModal();
+            Services.AddTransient<SettingsManager>();
+        }
+
+        [Fact]
+        public void SettingsPageShouldNotContainStopButtonIfScoutIsNotStartedWhenFirstOpened()
```

Comment:
![image](https://user-images.githubusercontent.com/56611129/144032234-469276d4-6580-4770-ae7a-a994f7ff2c9a.png)
 That long name